<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualAppMW</class>
 <widget class="QMainWindow" name="VisualAppMW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAMBADA Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="RenderWidget" name="OGL" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulator">
    <property name="title">
     <string>World</string>
    </property>
    <addaction name="actionSaveState"/>
    <addaction name="actionRestoreState"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="action_Pause"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuSimulator"/>
  </widget>
  <widget class="QStatusBar" name="sb"/>
  <action name="actionSaveState">
   <property name="text">
    <string>&amp;Save State</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRestoreState">
   <property name="text">
    <string>&amp;Restore State</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>render/RenderWidget.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
